ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml

```diff
@@ -695,6 +695,519 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cluster: api.ci
+  decorate: true
+  interval: 12h
+  labels:
+    ci.openshift.io/release-type: informing
+    job-env: remote-libvirt
+    job-release: "4.5"
+    job-test: e2e
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-origin-installer-e2e-remote-libvirt-s390x-4.5
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      - --kubeconfig=/etc/apici/kubeconfig
+      - --secret-dir=/usr/local/e2e-remote-libvirt-cluster-profile
+      - --secret-dir=/usr/local/pull-secret
+      - --target=e2e-remote-libvirt
+      - --template=/usr/local/e2e-remote-libvirt
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp/release:4.5
+      - name: BRANCH
+        value: "4.5"
+      - name: ARCH
+        value: s390x
+      - name: MULTIARCH_RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp-s390x/release-s390x:4.5
+      - name: BASTION_SVC_IP
+        value: 172.30.211.152
+      - name: BASTION_SVC_1_IP
+        value: 172.30.90.239
+      - name: BASTION_SVC_2_IP
+        value: 172.30.142.218
+      - name: BASTION_SVC_3_IP
+        value: 172.30.160.230
+      - name: BASTION_SVC_4_IP
+        value: 172.30.145.116
+      - name: BASTION_SVC
+        value: sshd.bastion-z.svc.cluster.local
+      - name: REMOTE_LIBVIRT_HOSTNAME
+        value: ocponz
+      - name: CLUSTER_TYPE
+        value: libvirt-s390x
+      - name: CONFIG_SPEC
+        value: |
+          resources:
+            '*':
+              limits:
+                memory: 4Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tag_specification:
+            cluster: https://api.ci.openshift.org
```

Comment:
You need to remove this line now, with it the job will never pass (the underlying tooling changed while this PR was in-flight).

Reply:
Thanks.
I guess I have to do the same for the 4.4 PR https://github.com/openshift/release/pull/8964?

Reply:
Yes